src/adapters/exec.stubs.ts

```diff
@@ -3,8 +3,8 @@ export type CreateStubExecSettings = {
     stdout: string;
 };
 
-export const createStubExec = ({ stderr = "", stdout = "" } = {}) =>
+export const createStubExec = ({ stderr = "", stdout = "" } = {}): jest.Mock =>
```

Comment:
🙁 I have a preference for the way it was before: not redundantly declaring return types. Could you please undo these unrelated source changes?

Reply:
Apologies for the change, I do remember updating the dependencies that triggered these changes due to TSLint 6.0.0.

Should be good if you can provide a PR to fix those TSLint issues while I revert most of the changes done 🙈